internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -82,6 +82,14 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) RequestBody(_ []byte, ope
 		bedrockReq.InferenceConfig.StopSequences = stopSequence
 	}
 
+	// Handle Anthropic vendor fields if present. Currently only supports thinking fields.
+	if openAIReq.AnthropicVendorFields != nil && openAIReq.Thinking != nil {
+		if bedrockReq.AdditionalModelRequestFields == nil {
+			bedrockReq.AdditionalModelRequestFields = make(map[string]interface{})
+		}
+		bedrockReq.AdditionalModelRequestFields["thinking"] = openAIReq.Thinking
```

Comment:
```suggestion
		bedrockReq.AdditionalModelRequestFields["thinking"] = openAIReq.AnthropicVendorFields.Thinking
```

Reply:
actually linter complains - i think its ok as is?